### PR TITLE
Fix OSX 10.11 build

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: co/connectionSet.cpp [hunk 1]
```diff
@@ -611,25 +611,26 @@ bool ConnectionSet::_setupFDSet()
     for( ConnectionsCIter i = _impl->allConnections.begin();
          i != _impl->allConnections.end(); ++i )
     {
-        ConnectionPtr connection = *i;
-        fd.fd = connection->getNotifier();
+        ConnectionPtr connectionPtr = *i;
+        const Connection& connection = *connectionPtr.get();
+        fd.fd = connection.getNotifier();
```
Comment (eile) [inline co/connectionSet.cpp]: ?

Comment (tribal-tec) [inline co/connectionSet.cpp]: New warning
https://github.com/sass/libsass/issues/1523

**Comment by tribal-tec** [comment]
retest this please

**Comment by tribal-tec** [comment]
good to merge?

**Pull request merged**